src/rpc/rawtransaction.cpp

```diff
@@ -1115,8 +1113,15 @@ static UniValue sendrawtransaction(const JSONRPCRequest& request)
     const uint256& hashTx = tx->GetHash();
 
     CAmount nMaxRawTxFee = maxTxFee;
-    if (!request.params[1].isNull() && request.params[1].get_bool())
-        nMaxRawTxFee = 0;
+    if (request.params[1].isBool()) {
+        if (request.params[1].get_bool()) nMaxRawTxFee = 0;
+    } else if (request.params[1].isNum()) {
+        size_t weight = GetTransactionWeight(*tx);
+        CFeeRate fr(AmountFromValue(request.params[1]));
+        nMaxRawTxFee = fr.GetFee((weight+3)/4);
```

Comment:
Might be good to add a quick comment here to clarify the `+3/4` is for rounding

Reply:
Added note.